src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/PackageModelFactory.cs

```diff
@@ -39,41 +39,42 @@ public PackageModel Create(PackageSearchMetadataContextInfo metadata, ContractIt
 
             EmbeddedResourcesCapability embeddedResources = new EmbeddedResourcesCapability(_packageFileService, metadata.Identity!, metadata.ReadmeUrl);
 
-            if (metadata.PackagePath != null)
+            // Transitive dependencies are only available in the Installed tab
+            if (metadata.TransitiveOrigins != null && itemFilter.Equals(ContractItemFilter.Installed))
             {
-                if (metadata.TransitiveOrigins != null)
-                {
-                    IVulnerableCapable vulnerableDatabaseCapability = new VulnerableDatabaseCapability(_packageVulnerabilityService, metadata.Identity);
-                    IDeprecationCapable noDeprecationCapable = new NoDeprecationCapability();
-                    return new TransitivelyReferencedPackageModel(
-                        metadata.Identity!,
-                        metadata.PackagePath,
-                        vulnerableDatabaseCapability,
-                        noDeprecationCapable,
-                        embeddedResources,
-                        metadata.TransitiveOrigins,
-                        metadata.Title,
-                        metadata.Description,
-                        metadata.Authors,
-                        metadata.ProjectUrl,
-                        metadata.Tags?.Split(','),
-                        metadata.OwnersList,
-                        metadata.DependencySets,
-                        metadata.Summary,
-                        metadata.Published,
-                        metadata.LicenseMetadata,
-                        metadata.LicenseUrl,
-                        metadata.RequireLicenseAcceptance,
-                        metadata.ReportAbuseUrl?.ToString(),
-                        metadata.IconUrl);
-                }
+                IVulnerableCapable vulnerableDatabaseCapability = new VulnerableDatabaseCapability(_packageVulnerabilityService, metadata.Identity);
+                IDeprecationCapable noDeprecationCapable = new NoDeprecationCapability();
+                return new TransitivelyReferencedPackageModel(
```

Comment:
Should we move this to the else case of when package path != null?

Are package paths always null for Transitive packages? If so, by having this code outside of the if check for if (metadata.PackagePath != null), it effectively becomes the "else" case. In its current state, the logic is confusing because we check for transitive, then we check for package path, then the else case for package path, followed by other criteria (if package is recommended).